fedmsg/commands/__init__.py

```diff
@@ -81,6 +81,9 @@ def _daemonize(self):
         except:
             from daemon.pidlockfile import PIDLockFile
 
+        msg = ('The use of the "--daemon" flag is deprecated and will be removed in '
+               'fedmsg-0.20.0. Use your init system to run fedmsg commands as daemons.')
+        warnings.warn(msg, category=UserWarning)
```

Comment:
Yeah, I was actually too lazy to run the tests locally to see if the coverage would go down so I just made a PR :(